rhel8/kickstart/ssg-rhel8-ospp-ks.cfg

```diff
@@ -0,0 +1,176 @@
+# SCAP Security Guide OSPP/USGCB profile kickstart for Red Hat Enterprise Linux 8
```

Comment:
One more note: I think USGCB for RHEL8 doesn't exist, so I prefer not to mention it here at all.